doc/sphinx/techref/plugins.rst

```diff
@@ -0,0 +1,77 @@
+Using FontForge Plugins
+=======================
+
+A FontForge plugin is a Python package that contains code to enhance
+the FontForge application in some way, most often by adding one or more
+entries to the Font View or Char View :doc:`Tools menu </ui/menus/toolsmenu>`.
+Using a plugin is as easy as installing the package (often with
+`pip <https:pypi.org/project/pip/>`_) and, when necessary, "configuring"
+the plugin. Almost any plugin of general interest will be listed in
+the `PyPI package database <https:pypi.org/search/?q=%22FontForge_plugin%22>`_.
```

Comment:
It's sad that there isn't proper support for custom tags (like you just have to have it as part of the description), but oh well, that's a pip/pypi limitation.

Reply:
I switched the new functions to Pascal case.